apps/dashboard/src/app/(dashboard)/user/[id]/edit-form.tsx

```diff
@@ -0,0 +1,53 @@
+import { type UserWrite, UserWriteSchema } from "@dotkomonline/types"
+import { createCheckboxInput, createSelectInput, createTextInput, useFormBuilder } from "../../../form"
+
+interface UseUserProfileWriteFormProps {
+  onSubmit(data: UserWrite): void
+  defaultValues?: Partial<UserWrite>
+  label?: string
+}
+
+export const useUserProfileEditForm = ({ defaultValues, onSubmit, label = "Bruker" }: UseUserProfileWriteFormProps) =>
+  useFormBuilder({
+    schema: UserWriteSchema,
+    onSubmit,
+    defaultValues,
+    label,
+    fields: {
+      firstName: createTextInput({
+        label: "Fornavn",
+        placeholder: "Ola",
+      }),
+      lastName: createTextInput({
+        label: "Etternavn",
+        placeholder: "Ola",
+      }),
+      phone: createTextInput({
+        label: "Telefon",
+        placeholder: "12345678",
```

Comment:
Dette eksempelet er ikke gyldig, og merk at for internasjonale må man ha med +xx